packages/tests/vitest.config.ts

```diff
@@ -16,6 +16,7 @@ export default defineConfig({
       include: ['*/src/**/*.{ts,tsx,js,jsx}'],
       exclude: ['**/deprecated/**'],
     },
+    // useAtomics: true,
```

Comment:
had to disable this @sachinraja

My computer panicked, might not be supported on M1 architecture or smth?

![CleanShot 2023-02-20 at 10 38 27](https://user-images.githubusercontent.com/51714798/220068659-69b7b979-1b01-4671-9409-bd913406daa9.png)

Reply:
that's what actually fixed this though 🤣